express/api/misc-utils.js

```diff
@@ -0,0 +1,4 @@
+function getCurrentTimestamp() {
```

Comment:
Does this need its own file? Why not include it in file-utils.js?

Reply:
I'll move it there. Was thinking it wasn't strictly 'file' related but my first thought was to put it there so I'll move it